File: CostExplorer/Domain/Bug.cs
namespace CostExplorer.Domain {
    public class Bug    {
        public string title { get; set; } 
        public string description { get; set; } 
        public int severity { get; set; } 
        public int environmentId { get; set; } 
        public int organizationId { get; set; } 
        public int systemId { get; set; } 
        public int stateId { get; set; } 
        public int id { get; set; } 
    }
}

File: notifications/models/bug.js
const Bug = (schema, types) => {
  return schema.define('bug', {
    id: {
      type: types.INTEGER
      , primaryKey: true
        //, autoIncrement: true  DISABLED BECAUSE THE ID COMES WITH THE BUG OBJECT BEING RECEIVED
    }
    , title: {
      type: types.TEXT
      , allowNull: false
    }
    , description: {
      type: types.TEXT
      , allowNull: false
    }
    , severity: {
      type: types.INTEGER
      , allowNull: false
    }
    , systemId: {
      type: types.INTEGER
      , allowNull: false
    }
    , environmentId: {
      type: types.INTEGER
      , allowNull: false
    }
    , organizationId: {
      type: types.INTEGER
      , allowNull: false
    }
    , userId: {
      type: types.INTEGER
      , allowNull: false
    }
    , stateId: {
      type: types.INTEGER
      , allowNull: false
    }
  })
}

module.exports = Bug
